### ISSUE TITLE
Reactable with multi-labels displayed when there is no filter

### ISSUE THREAD

**Issue by LittleBeannie** [elong0527/forestly#43]
The reactable generates ALL labels first and the select-list only filters the generated label. 
If nothing is selected in the select-list, the reactable display ae under ALL labels, including NONE, AEREL, AESER, AEACN, ect.. (Although these labels can be hid). 
However, we would like to only see the AE under NONE label when no filters are applied.

Two solutions:
- Set the default of the select-list as NONE;
- Generate a more interactive reactable: if you click AEREL, only durg-related AE is displayed, and if you click nothing, then nothing is displayed.

**Comment by elong0527** [comment]
Please try this 

```
library(crosstalk)
library(reactable)
library(crosstool)

shared_iris <- SharedData$new(iris)

bscols(
  
  filter_select("sepal_length", "Sepal Length", shared_iris, ~Species),
  
  reactable(shared_iris),

  crosstool(shared_iris, "transmitter",  init= which(iris$Species == "setosa")),
  
  widths = c(3,9,0)
)
```

Reference: 
https://github.com/bwlewis/crosstool
https://bwlewis.github.io/crosstool/
https://github.com/rstudio/crosstalk/issues/16

**Comment by LittleBeannie** [comment]
It works! Thanks~ a updated PR will be submitted later.

**Comment by LittleBeannie** [comment]
Hi Yilong, 

- I found there may be a bug: when we set `init=...`, it need to add `channel = ` and `reset = `. Otherwise, the filter will be applied to the initials. 

- In the example you provided, if you select `species = virginica`, there is no filtered output, because the filter is applied to your initials, and the initial are all `species = setosa`.

- Here is the fix of the bug:

`library(crosstalk)
library(reactable)
library(crosstool)

shared_iris <- SharedData$new(iris)

bscols(
  
  filter_select("sepal_length", "Sepal Length", shared_iris, ~Species),
  
  reactable(shared_iris),
  
  #crosstool(shared_iris, "transmitter",  init= which(iris$Species == "setosa")),
  crosstool(shared_iris, 
            # transceiver widgets are more like normal crosstalk widgets.
            class = "transceiver",  
            # set the initial value 
            init = which(iris$Species == "setosa"),
            # channel set to "filter" to use the crosstalk filter handle
            channel ="filter",
            # use with init = which(xx == yy) to reset the initial filter/select handle
            reset = rownames(iris)),
  
  widths = c(3,9,0)
)`


- Reference: the third example in https://bwlewis.github.io/crosstool/.

**Comment by LittleBeannie** [comment]
There is an argument `defaultSelected` in `reactable()`, which is a numeric vector of default selected row indices. 
I tried this argument but it doesn't work at all. After checking the source code of `reactable()` (https://github.com/glin/reactable/blob/master/R/reactable.R), I found the developers only throw warnings for `defaultSelected`. So I use `crosstool()` for the default display for the current version.
